pkg/nuclide/file-tree/spec/FileTreeController-spec.js

```diff
@@ -107,6 +107,23 @@ describe('FileTreeController', () => {
     });
   });
 
+  describe('openDuplicateDialog', () => {
+    it('gets called when triggering nuclide-file-tree:duplicate-selection', () => {
+      var el = React.findDOMNode(TestUtils.findRenderedDOMComponentWithClass(
+        fileTreeController._panelController.getChildComponent(),
+        'nuclide-file-tree'
+      ));
+
+      // Mock method
+      spyOn(fileTreeController, 'openDuplicateDialog');
+      atom.commands.dispatch(el, 'nuclide-file-tree:duplicate-selection');
+      expect(fileTreeController.openDuplicateDialog.calls.length).toBe(1);
+    });
+    xit('fires an Notification if the file could not be created', () => {
```

Comment:
Can you fill this in or delete it? The other dialog methods are also difficult to test, so deleting for now is fine if it's not easily testable.